articles/cloud-computing-cost-optimization-with-cloud-bursting/index.md

```diff
@@ -0,0 +1,73 @@
+
+When you need some extra space on your private cloud to, for example, run applications, you can always cloud burst to free up space. In this scenario, cloud bursting helps to transfer data from your private cloud to the public cloud for a period.
+
+#### Flexible scaling
+
+Organizations with fluctuating customer bases and those startups wanting to scale their operations rely on cloud bursting for rapid and convenient scaling. Cloud bursting offers an added convenience in that organizations do not need to predict demand usage manually. Besides, organizations can cloud burst to free up local resources and use them for other purposes.
+
+#### Business continuity
+
+Cloud bursting prevents critical applications from crashing; otherwise, operations could come to a complete standstill. Application crash prevention is very beneficial to organizations. It ensures organizations maintain their functions during high demand times.
+
+### Challenges of cloud bursting
+
+#### Managing configurations
+
+Organizations that cloud burst run the same resource types, for example, application servers, in multiple clouds. But choosing multiple cloud providers may bring some challenges in terms of different resource offerings. For instance, it may require an organization to manage and configure its applications and technology stacks across two environments.
+
+Differences may come in terms of cloud network configurations. Some cloud providers offer the concept of availability zones, while others do not. Still, some have security groups, others have Access Control Lists ([ACLs](https://geek-university.com/ccna/what-is-acl-access-control-list/#)) and [Subnets](https://en.wikipedia.org/wiki/Subnetwork#), and some have both.
+
+[Hypervisors](https://www.vmware.com/topics/glossary/content/hypervisor#), their features, and versions of hypervisors differ between cloud providers. Also, machine types and power vary from one cloud provider to another. Clouds also offer different storage types, including object, persistent, and block.
+
+Such differences may bring a downstream effect. For instance, for an organization to use different storage subsystems and hypervisors, it also has to use additional virtual machine images(https://www.informit.com/articles/article.aspx?p=1927741&seqNum=7) built for each cloud. Still, the organization has to maintain these base machine images with every security patch and operating system.
+
+#### Complexity and Latency
+
+Effective cloud bursting requires a flexible and scalable network that can handle the workload changes between public cloud infrastructures and on-premises private clouds. The management and [orchestration](https://www.mulesoft.com/resources/esb/what-application-orchestration#) of the applications moving between the two cloud environments must be in synchronization. And this requires two identical clouds with matching platforms and templates and a lot of scripting.
```

Comment:
Explanation for orchestration. In networking, orchestration has various meanings on various layers of the OSI model.